### ISSUE TITLE
Add numerical values to chapter filenames to sort them inside each module

### ISSUE THREAD

**Issue by Cmurilochem** [esciencecenter-digital-skills/research-software-support#47]
A new convension has been settled that the order of the chapters inside each module should be determined be numerical values at the start of each filename.
For example, inside the module `modules/publication`, files which were hitherto organized as:
```bash
├── discussion.md
├── slides-publication.pmd
├── software_publication_pipeline.md
├── automating_software_publication.md
├── ex_registries.md
├── reading.md
└── media
    ├── nlesc-dummy.png
    └── repository_logos.png
``` 
should now be named:

```bash
├── 01.discussion.md
├── 02.slides-publication.pmd
├── 03.software_publication_pipeline.md
├── 04.automating_software_publication.md
├── 05.ex_registries.md
├── 06.reading.md
└── media
    ├── nlesc-dummy.png
    └── repository_logos.png
```
The numerical values at the start of the filenames will determine their order of appearance within each module. We should then perform the proper adaptations to other modules accordingly.

**Comment by LourensVeen** [comment]
Are the slides supposed to be part of this sequence, or should they be considered a separate thing?

**Comment by Cmurilochem** [comment]
> Are the slides supposed to be part of this sequence, or should they be considered a separate thing?

Good question! I was asking right now this to myself. @JaroCamphuijsen or @raar1, could you clarify this ?

**Comment by LourensVeen** [comment]
I think I saw an issue somewhere about learning goals and learning methods, and I'd say the slides are for classroom teaching, while the text is for self study. So maybe it makes sense to treat those separately. On the other hand, the exercises will be in both.

**Comment by JaroCamphuijsen** [comment]
Yeah the slides document should be part of this sequence. so just `03.presentation.pmd` for example
